ng2-bootstrap.ts

```diff
@@ -16,3 +31,24 @@ export * from  './components/typeahead';
 export * from  './components/position'
 export * from  './components/common'
 export * from  './components/ng2-bootstrap-config';
+
+export default {
+  directives: [Dropdown, Progress, Tab, TabHeading],
+  components: [
+    Accordion,
+    AccordionPanel,
```

Comment:
So I believe all the `components` listed in this export should be in the `directives` collection altogether since they would in fact be part of the `directives` metadata in a user's source.